pages/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { useOutsideClick, useToggle } from '@/src/shared/hooks'
+import useWindowResize from '@/src/shared/hooks/useWindowResize'
+import ProductCard from '@/src/widgets/products-card/ProductCard'
+import Ranking from '@/src/widgets/ranking/Ranking'
+import SideMenu from '@/src/widgets/side-menu/SideMenu'
+import SideMenuMobile from '@/src/widgets/side-menu/SideMenuMobile'
+
+const home = () => {
```

Comment:
저희 컨벤션에 따라, 
```suggestion
export default function Home = () => {
```

이렇게 바꾸어야할 거 같아요!

Reply:
앗 습관때문에 ㅎㅎ 감사합니다!